blocks/v2-footer/v2-footer.css

```diff
@@ -0,0 +1,411 @@
+  }
+
+  .v2-footer .v2-footer-main-content{
+    display: grid; 
+    grid-template-columns: 1fr; 
+    grid-template-rows: 1fr 1fr 1fr 1fr; 
+    gap: 0 0; 
+    /* stylelint-disable-next-line declaration-block-no-redundant-longhand-properties */
+    grid-template-areas: 
+      "v2-logo"
+      "v2-content"
+      "v2-form"
+      "v2-social"; 
+      height: 70vh;
+  }
+  
+
+  .v2-social { 
+    grid-area: v2-social;
+    margin-top: 0px;
+  }
+
+  .v2-content { grid-area: v2-content; }
+
+  .v2-form { 
+    grid-area: v2-form;
+    height: 15vh;
+  }
+
+  .v2-logo { 
+    grid-area: v2-logo; 
+    height: 15vh;
+  }
+
+  @media (min-width: 495px) {
+    .v2-footer .footer-form-and-social-section {
+      width: 255px;
+    }
+  }
+  
+  @media (min-width: 1200px) {
+    .v2-footer {
+      padding-top: 96px;
+    }
+  
+    .v2-footer .v2-footer-content {
+      display: flex;
+      flex-flow: row wrap;
+    }
+  
+    .v2-footer .v2-footer-links-section {
+      display: flex;
+      justify-content: space-between;
+      align-items: flex-start;
+      gap: 20px;
+      flex-wrap: nowrap;
+    }
+
+    .v2-footer .v2-footer-links-section ul {
+      min-width: 0;
+      width:160px;
+    }
+
+    .v2-footer .v2-footer-links-section ul li{
+      margin-bottom: 14px;
+    }
+  
+    .v2-footer .v2-footer-links-col {
+      width: 185px;
+    }
+  
+    /* stylelint-disable-next-line no-descending-specificity */
+    .v2-footer .v2-footer-bottom-section li {
+      padding: 10px;
+      width: auto;
+    }
+  
+    .v2-footer .v2-footer-links-col-1 ul li {
+      width: 100%;
+    }
+  
+    .v2-footer .v2-footer-bottom-section {
+      width: 100%;
+      margin-top: 65px;
+    }
+  
+    .v2-footer .v2-footer-bottom-section ul {
+      width: 100%;
+      display: flex;
+      flex-wrap: nowrap;
+      max-width: unset;
+      align-items: flex-start;
+      justify-content: left;
+      margin-top: 22px;
+    }
+
+    /* stylelint-disable-next-line no-duplicate-selectors */
+    .v2-footer .v2-footer-main-content{
+      display: grid; 
+      grid-template-columns: 1fr 1fr 1fr; 
+      grid-template-rows: 1fr 1fr; 
+      gap: 0 0; 
+      /* stylelint-disable-next-line declaration-block-no-redundant-longhand-properties */
+      grid-template-areas: 
+      "v2-logo v2-content v2-form"
+      "v2-social v2-content v2-form";
+      height: 15vh;
+    }
+
+    .v2-social { 
+      margin-top: -80px;
+    }
+
+    .v2-footer .v2-footer-bottom-section-content {
+      display: flex;
+      align-items: end;
+      font-size: 12px;
+      justify-content: space-between;
+    }
+  }
+  
+  /* eloqua footer form styles - START */
+  .v2-footer .elq-form .field-style {
+    margin: 0;
+  }
+  
+  .v2-footer .elq-form .item-padding {
+    padding: 0;
+  }
+  
+  .v2-footer .eloqua-form .elq-form input[type="text"] {
+    background-color: var(--c-primary-white);
+    padding: 15px 10px;
+    border: none;
+    width: 200px;
```

Comment:
In the design for mobile, the form field is wider. There it’s 295px, but it should be 100% width minus the button. I think a `max-width: 320px` would be a good size.

Reply:
Updated